assets/scss/sections/_healthy-is-the-most-precious.scss

```diff
@@ -0,0 +1,39 @@
+.healthy-is-the-most-precious{
+    display: flex;
+    padding:50px 0;
```

Comment:
Czy to nie jest brak spójności w wielkości paddingów? Sprawdź z resztą sekcji

Reply:
Poprawione